.travis.yml

```diff
@@ -74,7 +74,7 @@ before_script:
 
 script:
   - conda list
-  - py.test -v --doctest-modules --doctest-ignore-import-errors -rs odo
```

Comment:
This is not how most people run their test suite so I would prefer to keep it as it is.